packages/benchmark/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@endo/benchmark",
+  "version": "0.1.0",
+  "private": true,
+  "description": "Endo benchmarking ",
+  "keywords": [],
+  "author": "Endo contributors",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/endojs/endo/tree/master/packages/skel#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/endojs/endo.git",
+    "directory": "packages/benchmark"
+  },
+  "bugs": {
+    "url": "https://github.com/endojs/endo/issues"
+  },
+  "type": "module",
+  "main": "./index.js",
+  "module": "./index.js",
+  "exports": {
+    ".": "./index.js",
+    "./package.json": "./package.json"
+  },
+  "scripts": {
+    "build": "exit 0",
+    "lint": "yarn lint:types && yarn lint:eslint",
+    "lint-check": "yarn lint",
+    "lint-fix": "yarn lint:eslint --fix && yarn lint:types",
+    "lint:eslint": "eslint '**/*.js'",
+    "lint:types": "tsc",
+    "postpack": "git clean -f '*.d.ts*'",
+    "prepack": "tsc --build tsconfig.build.json",
+    "test": "yarn rollup -c && eshost -h v8,xs dist/bundle.js",
+    "bench": "yarn rollup -c && node --jitless --cpu-prof dist/bundle.js"
```

Comment:
I am on the fence about whether my suggested name here should be changed to emphasize this is a node-only test whose purpose is to generate a `.cpuprofile` to get a flamegraph. Or whether we should leave it simple in expectation of generalizing the script to be cross platform using eshost. No change suggested.